dotnet/src/Agents/OpenAI/Internal/AssistantThreadActions.cs

```diff
@@ -459,10 +452,10 @@ private static async Task<IReadOnlyList<RunStep>> GetRunStepsAsync(AssistantClie
     {
         List<RunStep> steps = [];
 
-        await foreach (PageResult<RunStep> page in client.GetRunStepsAsync(run).ConfigureAwait(false))
+        await foreach (var step in client.GetRunStepsAsync(run).ConfigureAwait(false))
```

Comment:
This I can understand not supporting paging.  Can we please type the iteration variable?

```suggestion
        await foreach (RunStep step in client.GetRunStepsAsync(run).ConfigureAwait(false))
```